cmd/metrics-adapter/app/options/validation.go

```diff
@@ -0,0 +1,14 @@
+package options
+
+import (
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+)
+
+// Validate checks Options and return a slice of found errs.
+func (o *Options) Validate() error {
+	var errs []error
```

Comment:
Do you have other error handling later on?
If not how about use 
`return utilerrors.NewAggregate(o.CustomMetricsAdapterServerOptions.Validate())`
directly.

Reply:
ok, I got it.